lib/amazon/collector/ec2.rb

```diff
@@ -2,17 +2,15 @@ module Amazon
   class Collector
     module Ec2
       def source_regions(scope)
+        return [] unless scope[:region] == "us-east-1"
```

Comment:
This looks strange, why don't we want to get regions unless the region is us-east-1?  At least deserves a comment

Reply:
right, we have this in other places too (like flavors and volume types), I might need to abstract it to `default_region` or something

the thing is that using any region gives us the same result, so we want to do the API call in just 1 region

Reply:
done